docs/demo/new-bank-account-usecase.md

```diff
@@ -0,0 +1,80 @@
+# New Bank Account Use Case
+
+**Use Case**: As a user, I should be able to open a new bank account online by providing my digital 
+drivers license, digital drivers license assurance/evidence and digital credit report.
+
+## References
+- [Demo Recording](https://www.youtube.com/watch?v=JNUQaOwprT8&feature=youtu.be)
```

Comment:
I assume this link will need an update in a followup?

Reply:
Yes @troyronda .